src/visualization/utilities.jl

```diff
@@ -1126,7 +1126,7 @@ function mesh_plotting_wireframe(pds::PlotDataSeries{<:UnstructuredPlotData2D};
   @unpack x_face, y_face, face_data = pd
 
   if set_z_coordinate_zero
-    sol_f = zeros(eltype(first(face_data)), size(face_data)) # plot 2d surface by setting z coordinate to zero
+    sol_f = zeros(eltype(first(x_face)), size(x_face)) # plot 2d surface by setting z coordinate to zero
```

Comment:
Why this change?

Reply:
I set `face_data` as nothing for Plots.jl. It's used to set the z-coordinate for 3D surface plots, which aren't used for Plots.jl recipes.